ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -1,18 +1,30 @@
+#ptf --test-dir ptftests fast-reboot --qlen=1000 --platform remote -t 'verbose=True;dut_username="admin";dut_hostname="10.0.0.243";fast_reboot_limit=30;portchannel_ports_file="/tmp/portchannel_interfaces.json";vlan_ports_file="/tmp/vlan_interfaces.json";ports_file="/tmp/ports.json";dut_mac="4c:76:25:f5:48:80";default_ip_range="192.168.0.0/16";vlan_ip_range="172.0.0.0/22";arista_vms="[\"10.0.0.200\",\"10.0.0.201\",\"10.0.0.202\",\"10.0.0.203\"]"' --platform-dir ptftests --disable-vxlan --disable-geneve --disable-erspan --disable-mpls --disable-nvgre
 #
 #
-# This test measures length of DUT dataplane disruption in fast-reboot procedure.
+# This test checks that DUT is able to make FastReboot procedure
 #
 # This test supposes that fast-reboot initiates by running /usr/bin/fast-reboot command.
-# The test sequence are following:
-# 1. Check that DUT is stable. That means that pings work in both directions: from T1 to servers and from servers to T1.
+#
+# The test is using "pings". This is packets which are sent through dataplane in two directions
+# 1. From one of vlan interfaces to T1 device. The source ip, source interface, and destination IP are chosen randomly from valid choices. Number of packet is 100.
+# 2. From all of portchannel ports to all of vlan ports. The source ip, source interface, and destination IP are chosed sequentially from valid choices.
+#    Currently we have 500 distrinct destination vlan addresses. Our target to have 1000 of them.
+#
+# The test sequence is following:
+# 1. Check that DUT is stable. That means that "pings" work in both directions: from T1 to servers and from servers to T1.
 # 2. If DUT is stable the test starts continiously pinging DUT in both directions.
 # 3. The test runs '/usr/bin/fast-reboot' on DUT remotely. The ssh key supposed to be uploaded by ansible before the test
 # 3. As soon as it sees that ping starts failuring in one of directions the test registers a start of dataplace disruption
 # 4. As soon as the test sess that pings start working for DUT in both directions it registers a stop of dataplane disruption
-# 5. If the length of the disruption is less 30 seconds - the test passes
-
+# 5. If the length of the disruption is less 30 seconds (if not redefined by parameter) - the test passes
+# 6. If there're any drops, when control plane is down - the test fails
+# 7. When test start fast-reboot procedure it connects to all VM (which emulates T1) and starts fetching status of BGP and LACP
+#    if LACP is inactive on VMs and interfaces goes down test failes
+#    if BGP graceful restart timeout is not equal 120 seconds the test fails
+#    if BGP graceful restart is not enabled on DUT the test fails
+#    If BGP graceful restart timeout is less than 15 seconds the test fails
```

Comment:
>If BGP graceful restart timeout is less than 15 seconds the test fails [](start = 5, length = 70)

this is contradictory with the previous statement. 

"if BGP graceful restart timeout is not equal 120 seconds the test fails"